LoopCore/LoopCompletionFreshness.swift

```diff
@@ -12,7 +12,6 @@ public enum LoopCompletionFreshness {
     case fresh
     case aging
     case stale
-    case unknown
```

Comment:
Does this mean that a user's first impression of the Loop home screen, before they have gotten a loop, and everything is setup fine, they will see a red Loop?  That sounds wrong.  What is driving the removal of the unknown state?

Reply:
I repeated this question in #loop for those that made this decision can chime in. From the discussion, displaying a gray loop was frowned upon since the user is not made aware of this state through the on-boarding. It was decided that after setup but before loop starts loop, red was the proper colour to display.